src/features/nft/nftModals/bidBuyModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import { Button } from '../../../common/components/Buttons'
+import { InputNumberFormat } from '../../../common/components/Inputs'
+import { Info } from '../../../legacy/components/AppState'
+
+export type TBidBuyModal = {
+  username?: string
+  isOpen: boolean
+  handleClose?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const BidBuyModal = ({
+  username,
+  isOpen,
+  handleClose,
+}: TBidBuyModal): JSX.Element => {
+  const info = new Info()
+  const currencyName = info.testnet ? 'LSP' : 'PSL'
```

Comment:
I don't think this is the right way because it is already checked here https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/legacy/rpc.ts (Line 159). So you don't need to check it again. I think it should `const currencyName = info.currencyName`

Reply:
I am not sure how to get the currencyName. 
const currencyName = info.currencyName 
This doesn't work for me.

Reply:
@muhadovic Please take a look here 

<img width="1124" alt="1" src="https://user-images.githubusercontent.com/10944387/122729067-f7459700-d2a2-11eb-974a-069c7e64b227.png">
<img width="1127" alt="2" src="https://user-images.githubusercontent.com/10944387/122729084-fb71b480-d2a2-11eb-9f7c-a5344aedb6d1.png">

I can't add a patch file here for you, sorry.